config/channels/in-memory-channel/roles/dispatcher-clusterrole.yaml

```diff
@@ -76,8 +76,14 @@ rules:
       - eventing.knative.dev
     resources:
       - eventtypes
+      - eventpolicies
     verbs:
-      - create
       - get
       - list
       - watch
+  - apiGroups:
+      - eventing.knative.dev
+    resources:
+      - eventtypes
```

Comment:
is this related?

Reply:
Before my PR, it was 
```
  - apiGroups:
      - eventing.knative.dev
    resources:
      - eventtypes
    verbs:
      - create
      - get
      - list
      - watch
```
since we need the same (except for `create`) for the eventpolicies and they are in the same api group as eventtypes, I merged them and created one dedicated for eventtypes create: -->
```
  - apiGroups:
      - eventing.knative.dev
    resources:
      - eventtypes
      - eventpolicies
    verbs:
      - get
      - list
      - watch

  - apiGroups:
      - eventing.knative.dev
    resources:
      - eventtypes
    verbs:
      - create
```